administrator/components/com_associations/views/associations/view.html.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * View class for a list of articles.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class AssociationsViewAssociations extends JViewLegacy
+{
+	/**
+	 * An array of items
+	 *
+	 * @var  array
+	 */
+	protected $items;
+
+	/**
+	 * The pagination object
+	 *
+	 * @var  JPagination
+	 */
+	protected $pagination;
+
+	/**
+	 * The model state
+	 *
+	 * @var  object
+	 */
+	protected $state;
+	
+	/**
+	 * Display the view
+	 *
+	 * @param   string  $tpl  The name of the template file to parse; automatically searches through the template paths.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function display($tpl = null)
+	{
+		$app    = JFactory::getApplication();
+		$assoc = JLanguageAssociations::isEnabled();
+		
+		if ($assoc)
+		{
+			$this->state         = $this->get('State');
+			$this->filterForm    = $this->get('FilterForm');
+			$this->activeFilters = $this->get('ActiveFilters');
+
+			if (!$this->state->get('associationcomponent') == '' && !$this->state->get('associationlanguage') == '')
+			{
+				$this->items      = $this->get('Items');
+				$this->pagination = $this->get('Pagination');
+				$componentFilter  = $this->state->get('associationcomponent');
+				$parts            = explode('.', $componentFilter);
+				$comp             = $parts[0];
+				$assocItem        = $parts[1];
+
+				JHtml::addIncludePath(JPATH_ADMINISTRATOR . '/components/' . $comp . '/helpers/html');
```

Comment:
are we using any method from the component helpers here? isf not why loading them?